io/pci/pci_info_lsvpd.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Naresh Bannoth <nbannoth@linux.vnet.ibm.com>
+
+"""
+PCI device Test.
+Needs to be run as root.
+"""
+
+from avocado import Test
+from avocado.utils.software_manager import SoftwareManager
+from avocado import main
+from avocado.utils import pci
+
+
+class PciLsvpdInfo(Test):
+    '''
+    check lsvpd info
+    '''
+    def setUp(self):
+        '''
+        To check and install dependencies for the test
+        '''
+        sm = SoftwareManager()
+        for pkg in ["lsvpd"]:
+            if not sm.check_installed(pkg) and not sm.install(pkg):
+                self.skip("%s package is need to test" % pkg)
+
+    def test_pcilsvpdinfo(self):
+        error = []
+        for i in pci.get_pci_addresses():
```

Comment:
just want to add one more thing please use variable name as meaningful as `i`  not looks good